<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-5HBRUB1" failures="2" tests="2" name="Default test(failed)" time="0.182" errors="0" timestamp="2021-03-31T19:06:05 IST">
  <testcase classname="ToRun_FailTestCases.RunOnlyFailTestCases" name="sample3" time="0.015">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at ToRun_FailTestCases.RunOnlyFailTestCases.sample3(RunOnlyFailTestCases.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- sample3 -->
  <testcase classname="ToRun_FailTestCases.RunOnlyFailTestCases" name="sample4" time="0.003">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at ToRun_FailTestCases.RunOnlyFailTestCases.sample4(RunOnlyFailTestCases.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- sample4 -->
  <testcase classname="ToRun_FailTestCases.RunOnlyFailTestCases" name="sample1" time="0.0">
    <ignored/>
  </testcase> <!-- sample1 -->
  <testcase classname="ToRun_FailTestCases.RunOnlyFailTestCases" name="sample5" time="0.0">
    <ignored/>
  </testcase> <!-- sample5 -->
  <testcase classname="ToRun_FailTestCases.RunOnlyFailTestCases" name="sample2" time="0.0">
    <ignored/>
  </testcase> <!-- sample2 -->
</testsuite> <!-- Default test(failed) -->
